xios_examples/context_test/custom_type_mod.F90

```diff
@@ -0,0 +1,15 @@
+module custom_type_mod
+
```

Comment:
a code comment explaining what this module is here for would be really helpful please